src/vcpkg-test/configmetadata.cpp

```
@@ -93,6 +94,10 @@ TEST_CASE ("config without ce metadata", "[ce-metadata]")
             "kind": "filesystem",
             "path": "path/to/registry",
             "packages": [ "zlib" ]
+        },
+        {
+            "kind": "artifact",
+            "location": "https://github.com/microsoft/vcpkg-artifacts"
```

This should fail to parse as it's missing "name"